src/NATS.Client.KeyValueStore/Internal/NatsKVWatcher.cs

```diff
@@ -383,10 +383,14 @@ private async ValueTask<INatsJSConsumer> CreatePushConsumer(string origin)
             config.HeadersOnly = true;
         }
 
-        if (sequence > 0)
+        // Resume from a specific revision ?
+        if (sequence > 0 || _opts.ResumeAtRevision > 0)
```

Comment:
do we still need to do these if we assigned `_sequenceStream = _opts.ResumeAtRevision` in the constructor?

Reply:
Ah! you're spot on, apologies. can we simplify the logic so sequence only checked once? e.g.:
```csharp
if (sequence > 0)
{
    config.DeliverPolicy = ConsumerConfigDeliverPolicy.ByStartSequence;
    config.OptStartSeq = sequence + 1;
}
else if (_opts.ResumeAtRevision > 0)
{
    config.DeliverPolicy = ConsumerConfigDeliverPolicy.ByStartSequence;
    config.OptStartSeq = _opts.ResumeAtRevision;
}
```